etl/scripts-ccao-data-warehouse-us-east-1/sale/sale-flag_override.R

```diff
@@ -0,0 +1,44 @@
+library(readxl)
+library(janitor)
+library(dplyr)
+library(arrow)
+library(aws.s3)
+source("utils.R")
+
+df <-
+  read_excel(
+    "O:/CCAODATA/data/sale/res_characteristic_potentials_reviewed.xlsx" # nolint
+  ) %>%
+  janitor::clean_names() %>%
```

Comment:
Handled this ingest with [this comment](https://github.com/ccao-data/data-architecture/issues/946#issuecomment-3657549627) in mind. I believe this path will work out of the box on everyone's local machine, but I am not sure.